es_source.go

```diff
@@ -87,6 +87,25 @@ func NewElasticsearchSource(config *ElasticsearchConfig) (Source, error) {
 	}, nil
 }
 
+// Create a new ElasticsearchSource from the input object, but add extra SourceFields
+// to include to the new ElasticsearchSource instance.
+func (e *ElasticsearchSource) addExtraFields(extraFields map[string]string) *ElasticsearchSource {
+	sourceFields := make(map[string]string)
+	for k, v := range e.SourceFields {
+		sourceFields[k] = v;
```

Comment:
Go formatting looks a bit off: normal line-ending is just newline-delimited without any semi-colons. I'd recommend just running `make format` or `go fmt ./...` from the project directory to automatically apply standard formatting.

Reply:
Thanks! -- Ran sources through the go pretty-printer and will update PR.